yucca/training/augmentation/YuccaAugmentationComposer.py

```diff
@@ -25,14 +25,24 @@
 
 
 class YuccaAugmentationComposer:
+    """
+
+    Deep Supervision:
+        When enabled it must also be applied in the validation transforms, since the DeepSupervisionLoss (which is a wrapper around a regular loss)
+        expects a LIST of ground truth tensors rather than the raw ground truth tensor. Since the factors used in the
+        val_transforms is only (1., ) it simply wraps the tensor in a list.
+    """
+
```

Comment:
Note to self: remove this documentation again (you're brilliant and you figured out a way to avoid this)